### PR TITLE
Fix: Adding 5 min to the reconciliation loop for apply_controller.go

### PR DESCRIPTION
**Pull request by aagusuab**
### Description of your changes

Fixes #
https://github.com/Azure/k8s-work-api/issues/82
I have:

- [x] Read and followed Caravel's [Code of conduct](https://github.com/Azure/k8s-work-api/blob/master/code-of-conduct.md).
- [x] Run `make reviewable` to ensure this PR is ready for review.

### CODE REVIEW

File: pkg/controllers/apply_controller.go [hunk 1]
```diff
@@ -166,7 +167,7 @@ func (r *ApplyWorkReconciler) Reconcile(ctx context.Context, req ctrl.Request) (
 		utils.EventReasonReconcileComplete,
 		utils.MessageManifestApplyComplete)
 
-	return ctrl.Result{}, nil
+	return ctrl.Result{RequeueAfter: time.Minute * 5}, nil
```
Comment (Arvindthiru) [inline pkg/controllers/apply_controller.go:170]: Do we have a test to check if the member cluster does get synced after 5 mins is done ?. This is probably an IT

Comment (aagusuab) [reply pkg/controllers/apply_controller.go:170]: I checked it by deploying the controller, it did sync after 5 minutes.

Comment (Arvindthiru) [reply pkg/controllers/apply_controller.go:170]: Please add something similar to this https://github.com/Azure/fleet/blob/main/pkg/controllers/internalmembercluster/member_controller_integration_test.go#L109

Comment (aagusuab) [reply pkg/controllers/apply_controller.go:170]: Added a UT.

**Review by Arvindthiru — APPROVED** [review]

**Pull request merged**